src/i18n/en.js

```diff
@@ -94,7 +95,13 @@ You'll find an Email with steps to follow.`,
     copyright_MD: `© ${year} Doppler LLC. All rights reserved. [Privacy Policy & Legals](${urlPrivacyFromLogin}).`,
     enter_doppler: `Log In`,
     enter_doppler_sub: `Today is a good day to boost your business with Email, Automation & Data Marketing!`,
-    error_payment_MD: `Account blocked, please [contact Support](${mailtoSupport}).`,
+    error_payment_MD_offline: `
+Ouch! This account has been suspended for non-payment.
+
+Please [contact the Customer Support team](${mailtoSupportBlockedAccount}) for help.`,
+    error_payment_online_link: `Chat with the Customer Support team`,
+    error_payment_online_part1: `Ouch! This account has been suspended for non-payment. `,
+    error_payment_online_part2: ` for help.`,
```

Comment:
It does not look fine, there should be a way to inject the button, like this:

```suggestion
    error_payment_online_part1: `Ouch! This account has been suspended for non-payment. {clickeableMessage} for help.`,
```

But, if you have no time to research about it now, please add a TODO note to make it clear that we do not want to deal with this kind of things in this way in the future:

```suggestion
    // Ugly patch because I have no time to research how to inject an element.
    // TODO: merge these two sections in a single entry.
    error_payment_online_part1: `Ouch! This account has been suspended for non-payment. `,
    error_payment_online_part2: ` for help.`,
```

Reply:
Creo que la solución anda por acá: https://formatjs.io/docs/react-intl/components#rich-text-formatting

Si no me equivoco @fcoronelmakingsense estuvo jugando con esas opciones.